code/failpoint.go

```diff
@@ -60,7 +60,11 @@ func (fp *Failpoint) hdr(varname string) string {
 		varname = "_"
 	}
 	return hdr + varname + ", __fpTypeOK := v" + fp.name +
-		".(" + fp.varType + "); if !__fpTypeOK { goto __badType" + fp.name + "} "
+		".(" + fp.varType + "); if !__fpTypeOK { goto __badType" + fp.name + "}"
+}
+
+func (fp *Failpoint) wait() string {
+	return "/* gofail-wait */ " + fp.Runtime() + ".Wait()"
```

Comment:
I think `pause` is better than `wait`. See https://www.freebsd.org/cgi/man.cgi?query=fail&sektion=9&apropos=0&manpath=FreeBSD%2B10.0-RELEASE